src/App.jsx

```diff
@@ -1,13 +1,155 @@
+                    <span>{sum.kiwi}</span>
+                    <button type="button" onClick={() => addButton("kiwi")}>+</button>
+                </Button>
+                <div>
+                    <button type="button" className="reset-button" onClick={handleReset}>Reset</button>
+                </div>
+            </main>
+            <footer>
+                <form onSubmit={handleSubmit}>
+                    <label htmlFor="first-name">Voornaam:</label>
+                    <input type="text" name="firstName" id="first-name" value={formState.firstName}
+                           onChange={handleChange}/>
+                    <br/>
+                    <label htmlFor="last-name">Achternaam:</label>
+                    <input type="text" name="lastName" id="last-name" value={formState.lastName}
+                           onChange={handleChange}/>
+                    <br/>
+                    <label htmlFor="age">Leeftijd:</label>
+                    <input type="number" name="age" id="age" value={formState.age} onChange={handleChange}/>
+                    <br/>
+                    <label htmlFor="postal-code">Postcode</label>
+                    <input type="text" name="postalCode" value={formState.postalCode} id="postal-code"
+                           onChange={handleChange}/>
+                    <br/>
+                    <label htmlFor="deliver-freq">Bezorgfrequentie</label>
+                    <select id="deliver-freq" name="deliverFrequency" value={formState.deliverFrequency}
+                            onChange={handleChange}>
+                        <option value={"iedere week"}>iedere week</option>
+                        <option value={"om de week"}>om de week</option>
+                        <option value={"iedere maand"}>iedere maand</option>
+                    </select>
+                    <br/>
+                    <label>
+                        <input type="radio" name="timeSlot" value="Overdag" checked={formState.timeSlot === "Overdag"}
+                               onChange={handleChange}/>
+                        Overdag</label>
+                    <label>
+                        <input type="radio" name="timeSlot" value="'s Avonds"
+                               checked={formState.timeSlot === "'s Avonds"} onChange={handleChange}/>
+                        s Avonds</label>
+                    <br/>
+                    <label htmlFor="remark">Opmerking:</label>
+                    <textarea id="remark" name="remark" value={formState.remark} onChange={handleChange}></textarea>
+                    <br/>
+                    <label htmlFor="terms">
+                        <input type="checkbox" id="terms" name="terms" checked={formState.terms}
+                               onChange={handleChange}/>
+                        Ik ga akkoord met de voorwaarden
+                    </label>
+                    <br/>
```

Comment:
`<br/>` zijn niet heel netjes om te gebruiken, hier gebruiken we margin en padding voor :)